src/store/reducers/albumReducer.js

```diff
@@ -94,13 +92,13 @@ const albumReducer = (state = initialState, action) => {
     case `${SEARCH_ALBUM_LASTFM}_FULFILLED`:
       return {
         ...state,
-        list: fetchLastfmAlbums(get(action.payload, 'data', [])),
+        list: action.payload,
       };
 
     case `${SEARCH_ALBUM_DISCOGS}_FULFILLED`:
       return {
         ...state,
-        list: fetchDiscogsAlbums(get(action.payload, 'data', [])),
+        list: action.payload,
```

Comment:
With this change, I should be able to unify both `SEARCH_ALBUM_{PROVIDER}` into a single `SEARCH_ALBUM` action type

Reply:
Like I did in https://github.com/elamperti/OpenWebScrobbler/pull/197/commits/5ad1ec8bfca6c9f2a3a84bd1f9064f14274ea1f5